### ISSUE TITLE
Buttons getting overlapped in 175% zoom in Chrome

### ISSUE THREAD

**Issue by ieshaan12** [jaykay12/CricAI#22]
![image](https://user-images.githubusercontent.com/31623825/96103098-e44d1c80-0ef4-11eb-81c3-a48a785cb938.png)

I was just testing it out, and found this, I'd be glad to work if that's fine.

**Comment by jaykay12** [comment]
@ieshaan12 Yes, sure. You can propose a fix for this, I will be delighted to merge the PR.

**Comment by ieshaan12** [comment]
@jaykay12 Check it out.
